docs/source/models/efficientnet.rst

```diff
@@ -0,0 +1,31 @@
+EfficientNet
+============
+
+.. currentmodule:: torchvision.models
+
+The EfficientNet model is based on the `EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks <https://arxiv.org/pdf/1905.11946.pdf>`__
```

Comment:
Let's link to the arxiv page instead of linking to the `.pdf` file, because this can trigger undesired downloads

Reply:
That's a good point indeed. :ok_hand: